modules/cloudwatch_too_many_errors/main.tf

```diff
@@ -0,0 +1,28 @@
+terraform {
+  required_version = ">= 1.0.0"
+
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = ">= 3.0.0"
+    }
+  }
+}
+
+resource "aws_cloudwatch_metric_alarm" "this" {
+  metric_name         = "logback.events.count"
```

Comment:
Denne vil kun være spesifikt for logback/java applikasjoner? Kanskje legge ved noe info om det i README for sikkerhetsskyld?

Reply:
Ja, jeg tar i tillegg en sjekk på om det er en bedre måte å gjøre det på. Dette er gjenbruk fra en alarm vi hadde en i tidligere ecs modul som vi ønsket å ha videre

Reply:
Vi får disse fordi vi har satt opp micrometer med Micronaut, ja

Reply:
Ja, dette vil være for spesifikt. Det blir jo ikke bare Java spesifikt, men logback specific 😅

Her ville jeg heller ha laget en log metric så det kan brukes av andre og!